src/main/resources/maps/testGridMap/mapsettings.cmf

```diff
@@ -0,0 +1,12 @@
+#
+# Test map layout 
+#
+STARTROOM: startroom
+ENDROOM: endroom
+NOROOMS: 6
+room1
```

Comment:
Since the room reading format should be changed later this should not be doen this sprint. Room sizes should not have to be defined in the name of the file as they are all ready defined in the file itself. I made a new issue so we can address this in a later sprint, see #404 .